src/explorer/test/test.jl

```diff
@@ -0,0 +1,6 @@
+##  Mimi UI
+using Mimi
+
+## Test 1:  One Region Model
+include(joinpath(@__DIR__, "01-one-region-model/one-region-model.jl"))
+explore(my_model)
```

Comment:
Same comment as above, it strikes me that we should delete everything (or move somewhere else) that is in ``src/explorer/test``.